vector/src/main/java/im/vector/app/features/login/AbstractLoginFragment.kt

```diff
@@ -69,6 +69,11 @@ abstract class AbstractLoginFragment : VectorBaseFragment(), OnBackPressed {
     }
 
     override fun showFailure(throwable: Throwable) {
+        // Only the resumed Fragment can eventually show the error, to avoid multiple dialog display
+        if (!isResumed) {
```

Comment:
Yes I guess it's ok to do that. It's the issue when you listen to the same things on several fragments...